README.md

```diff
@@ -31,17 +31,19 @@ See the section on <a href="#safety">safety</a> below for details of known unsaf
 
 ## Supported Platforms
 
-We aim to support *at least* Active LTS and Current Node.js releases. `leveldown` ships with prebuilt binaries for [many platforms](https://github.com/Level/leveldown/releases) and is known to work on:
+We aim to support *at least* Active LTS and Current Node.js releases, as well as any future Node.js release thanks to [N-API](https://nodejs.org/api/n-api.html). Because N-API has an experimental status in node 6 and early 8.x releases, the minimum node version for `leveldown` is `8.6.0`.
+
+The `leveldown` npm package ships with prebuilt binaries for popular 64-bit platforms and is known to work on:
 
 * **Linux** (including ARM platforms such as Raspberry Pi *and Kindle!*)
 * **Mac OS**
 * **Solaris** (SmartOS & Nodejitsu)
 * **FreeBSD**
 * **Windows**
 
-When installing `leveldown`, [`prebuild-install`](https://github.com/prebuild/prebuild-install) will install prebuilt binaries from GitHub if they exist and fallback to a compile step if they don't. In that case you'll need a [valid `node-gyp` installation](https://github.com/nodejs/node-gyp#installation).
+When installing `leveldown`, [`node-gyp-build`](https://github.com/mafintosh/node-gyp-build) will check if a compatible binary exists and fallback to a compile step it it doesn't. In that case you'll need a [valid `node-gyp` installation](https://github.com/nodejs/node-gyp#installation).
 
-If you don't want to use the prebuilt binary for the platform you are installing on, specify the `--build-from-source` flag when you install. If you are working on `leveldown` itself and want to re-compile the C++ code it's enough to do `npm install`.
+~~If you don't want to use the prebuilt binary for the platform you are installing on, specify the `--build-from-source` flag when you install.~~ If you are working on `leveldown` itself and want to re-compile the C++ code it's enough to do `npm install`.
```

Comment:
Not sure about this. Does `node-gyp-build` (the CLI) support `--build-from-source`?

Reply:
No, it doesn't seem like it.

Reply:
@vweevers Think we should patch this as well?

Reply:
People on alpine may need it, assuming https://github.com/Level/leveldown/issues/388 also applies to prebuildify

Reply:
Update list of actions.